packages/terra-disclosure-manager/docs/README.md

```diff
@@ -0,0 +1,209 @@
+|`dismissPresentedComponent`|A function that pops the currently disclosed component off the disclosure stack.|
+|`closeDisclosure`|A function that closes the disclosure and removes all components from the disclosure stack.|
+
+`children` Object API:
+
+|Key|Value|
+|---|---|
+|`components`|An Array of React components to render as face-up content. These components were updated with an AppDelegate prop with DisclosureManager integration.|
+
+`disclosure` Object API:
+
+|Key|Value|
+|---|---|
+|`components`|An Array of React components to render in a disclosure mechanism. These components were updated with an AppDelegate prop with DisclosureManager integration.|
+|`isOpen`|A boolean indicating the current display state of the DisclosureManager.|
+|`isFocused`|A boolean indicating the current focus state of the DisclosureManager.|
+|`isMaximized`|A boolean indicating the current maximize state of the DisclosureManager.|
+|`size`|The String size of the disclosure.|
+
+Example (using the Modal and SlideGroup):
+```javascript
+<DisclosureManager
+  supportedDisclosureTypes={['modal']}
+  render={(manager) => (
+    <div>
+      {manager.children.components} // These are the Child1 and Child2 components provided as children to the DisclosureManager.
+      <Modal
+        isFocused={manager.disclosure.isFocused}
+        isOpen={manager.disclosure.isOpen}
+        onRequestClose={() => {
+          manager.closeDisclosure();
+        }}
+        closeOnEsc
+        closeOnOutsideClick
+      >
+        <SlideGroup items={manager.disclosure.components} isAnimated />
+      </Modal>
+    </div>
+  )}
+>
+  <Child1 />
+  <Child2 />
+</DisclosureManager>
+
+```
+
+### Interacting with the Disclosure Manager
+
+#### Children
+
+The AppDelegate provided to the child components contains a `disclose` function. This `disclose` function validates the disclosure type with which it is provided against the set of supported disclosure types given to the DisclosureManager as a prop. If the provided type is not supported, and if the DisclosureManager was given an AppDelegate prop to fall back to, the DisclosureManager will call the disclose function provided by its AppDelegate prop.
+
+If the type is supported, the DisclosureManager will check the currently disclosured content's state to ensure it can be replaced. If the disclosure is denied, then `disclose` returns a rejected Promise. If the disclosure is allowed, then a resolved Promise is returned. This Promise will be resolved with an Object containing functions and Promises that can be used to manipulate the disclosure, if necessary. Included are `dismissDisclosure`, a function that will dismiss the disclosed content, as well as `afterDismiss`, a deferred Promise that will be resolved when the disclosed content is dismissed by any means. Alternatively, if the additional logic isn't needed, the returned Promise can be completely ignored.
```

Comment:
currently _disclosed_ ~content's~ components state

Reply:
> This Promise will be resolved with an Object containing functions and Promises that can be used to manipulate the disclosure, if necessary.

I am a little unclear on this statement. Must this be provided or is this how the DisclosureManger is handling it.

Reply:
Included ~are~ is `dismissDisclosure`